### DISCUSSION TITLE
Implementing sticky notes?

### DISCUSSION THREAD

**Discussion by fralau** [markmap/markmap#198] [General]
Is there a way to implement **sticky notes** (on nodes) in markmind? In essence a sticky note would be represented in the text as markdown; and on the diagram as an icon; hovering on it or clicking on it would open the content.

I believe that might be an interesting feature for inserting lots of detail information that should be displayed only on demand?

A question would be: what markdown construct to use in the text?

A possibility might be: in any header or list item, use the `NOTE:` (uppercase string), eg.

```markdown
## List of tasks

  - First
  - Second task #NOTE:  This is a sticky note.
  - Third  task
    NOTE: This is another sticky note.
```
What would you think?

**Comment by MathishaN** [comment]
One solution is to put the note's content in a MarkDown file. Then link to it from the node.

```

## List of tasks

  - First
  - Second task [NOTE](note01.md).
  - Third  task [NOTE](note02.md).

```
Then there will be a lot more room for detailed notes without cluttering the map itself.

**Reply by MathishaN** [reply]
Another option is to use the 
`<!-- markmap: fold -->` option.

Example

```
# heading
- First node
- Second node (see note) <!-- markmap: fold -->
	- Note text goes here. It's collapsed but can be expanded in there rendered view

```
